platformio.ini

```diff
@@ -477,21 +478,51 @@ debug_init_break = tbreak setup
 board_build.partitions = config/partitions_custom_noota.csv
 
 ;===============
-
-[env:mesmerizer]
-extends         = dev_mesmerizer
+;
+[mesmerizer_common]
 build_flags     = -DMESMERIZER=1
                   -DUSE_HUB75=1
                   -DSHOW_VU_METER=1
                   -O3
-                  ${psram_flags.build_flags}
-                  ${remote_flags.build_flags}
-                  ${dev_mesmerizer.build_flags}
-lib_deps        = ${dev_mesmerizer.lib_deps}
-                  ${matrix_config.lib_deps}
-board_build.embed_files = ${env.board_build.embed_files}
-                          ${matrix_config.board_build.embed_files}
-debug_tool      = esp-prog
+                   ${psram_flags.build_flags}
+                   ${remote_flags.build_flags}
+                   ${dev_mesmerizer.build_flags}
+lib_deps         = ${dev_mesmerizer.lib_deps}
+                   ${matrix_config.lib_deps}
+; Until I can refactor the base libs, at least quit fetching/compiling these.
+lib_ignore       = Adafruit ILI9341
+                   ; Bounce2 - lost that argument
+                   Bodmer/TJpg_Decoder
+                   U8g2
+                   fastled/FastLED
+                   plageoj/UrlEncode
+
+embed_files      = ${env.board_build.embed_files}
+                   ${matrix_config.board_build.embed_files}
+debug_tool       = esp-prog
+
+[env:mesmerizer]
+extends         = dev_mesmerizer
+build_flags     = ${mesmerizer_common.build_flags}
+lib_deps        = ${mesmerizer_common.lib_deps}
+lib_ignore      = ${mesmerizer_common.lib_ignore}
+board_build.embed_files = ${mesmerizer_common.embed_files}
+debug_tool      = ${mesmerizer_common.debug_tool}
+
+; An additional mesmerizer build that includes more effects, but that weren't
+; cool enough to make the default build. They will appear in the web interface
+; as disabled by default, so turn on the ones you like.
+
+[env:mesmerizer_all]
+extends         = dev_mesmerizer
+build_flags     = ${mesmerizer_common.build_flags}
+		  -D USE_OPTIONAL_EFFECTS=1
+lib_deps        = ${mesmerizer_common.lib_deps}
+lib_ignore      = ${mesmerizer_common.lib_ignore}
+board_build.embed_files = ${mesmerizer_common.embed_files}
+debug_tool      = ${mesmerizer_common.debug_tool}
+
+;==============
 
 [env:mesmerizer_lolin]
```

Comment:
If we have a mesmerizer_common (merged with matrix_config or not), then I think we should use it for all mesmerizer environments, including this one.